test/test_responses.py

```diff
@@ -0,0 +1,19 @@
+from responses import GoodMorning
+
+# test instance of good morning class- response to a message
+gm = GoodMorning()
+
+# Positive tests in each of the cases the bot should recognise this as a valid good morning message and react with a hot drink emoji
+def test_good_morning():
+    assert gm.good_morning("Добро утро!")
+    assert gm.good_morning("ДОБРО УТРО!")
+    assert gm.good_morning("Лек ден и добро утро приятели!")
+    assert gm.good_morning("ДоБрО УтРо.")
+
+# Negative test cases in each of them the bot should identify that this is not a good morning message and therefore not react to it
+# with a hot drink emoji
+def test_not_good_morning():
```

Comment:
great, could we test `respond` method too?

Reply:
There is no need for explicit comments - an import is an import. After all it is not a python tutorial.

Reply:
Agreed the comments may be way too extensive and unnecessary, I am removing them.

Reply:
It is hard for me to understand how to simulate a message object for testing purposes.

Reply:
You can check how to mock with your library of choice for testing.